javascriptv3/example_code/cognito/README.md

```diff
@@ -17,18 +17,48 @@ Amazon Cognito user pools let you add registration and sign-in to your apps.
 
 ## Code examples
 
+### Single actions
+
+Code excerpts that show you how to call individual service functions.
+
+- [Sign up a user](./actions/sign-up.js) (SignUp)
+- [Get information about a user](./actions/admin-get-user.js) (AdminGetUser)
+- [Resend a confirmation code](./actions/resend-confirmation-code.js) (ResendConfirmationCode)
+- [Confirm a user](./actions/confirm-sign-up.js) (ConfirmSignUp)
+- [List users](./actions/list-users.js) (ListUsers)
+- [Start authentication with administrator credentials](./actions/admin-initiate-auth.js) (AdminInitiateAuth)
+- [Get a token to associate an MFA application with a user](./actions/associate-software-token.js) (AssociateSoftwareToken)
+- [Verify an MFA application with a user](./actions/verify-software-token.js) (VerifySoftwareToken)
+- [Respond to an authentication challenge](./actions/admin-respond-to-auth-challenge.js) (AdminRespondToAuthChallenge)
+- [Confirm an MFA device for tracking](./actions/confirm-device.js) (ConfirmDevice)
+- [Start authentication with a tracked device](./actions/initiate-auth.js) (InitiateAuth)
+- [Respond to SRP authentication challenges](./actions/respond-to-auth-challenge.js) (RespondToAuthChallenge)
+
 ### Scenarios
 
 Code examples that show you how to accomplish a specific task by calling multiple functions within the same service.
 
 - [Use a custom authentication flow](./scenarios/lambda-triggers)
+- [Sign up a user with a user pool that requires MFA](./scenarios/basic)
```

Comment:
Same note about alphabetical order.

Reply:
We probably want the MVP at the top, and the rest can be alphabetical. In this particular case, those are the same thing :) I'll swap them.